code/src/components/app-typography/AppTypography.tsx

```diff
@@ -13,7 +13,12 @@ type TextVariant =
   | "caption-small";
 type AppTypographyVariant = HeadingVariant | TextVariant;
 
-type AppTypographyProps = Omit<TypographyProps, "variant" | "children"> & {
+type BaseComponent = React.ElementType<any, keyof React.JSX.IntrinsicElements>;
```

Comment:
Please use just `ElementType`, do not use `React.`. If you want to use `React.`, just import it from `react` package

Reply:
Thank, fixed